src/coreclr/jit/fgbasic.cpp

```diff
@@ -662,6 +662,8 @@ void Compiler::fgReplaceJumpTarget(BasicBlock* block, BasicBlock* oldTarget, Bas
 
                 if (block->FalseEdgeIs(oldEdge))
                 {
+                    // Branch was degenerate, simplify it first
+                    //
                     // fgRemoveRefPred returns nullptr for BBJ_COND blocks with two flow edges to target
```

Comment:
Since you're here, could you please remove this comment about `fgRemoveRefPred` returning nullptr? The new version of it that takes in a flow edge doesn't return anything. Thanks!

Reply:
Yep.